vms/registration/passwordmatch.py

```diff
@@ -0,0 +1,6 @@
+# account can only be created if both passwords match
+def match_password(password1, password2):
```

Comment:
IMO lets name it `utils.py`, so that any future utilities of registration app will be added here.